### PR TITLE
Bugfix/jinja2 change delimiters of the template folder

### PR DESCRIPTION
**Pull request by mariovagomarzal**
Addresses https://github.com/cookiecutter/cookiecutter/issues/1736.

Based on the bugfix by @liortct in https://github.com/cookiecutter/cookiecutter/pull/1767 but using the organization name throughout the project.

### CODE REVIEW

File: cookiecutter/find.py [hunk 1]
```diff
@@ -3,21 +3,28 @@
...
     """
-    logger.debug('Searching %s for the project template.', repo_dir)
+    logger.debug("Searching %s for the project template.", repo_dir)
 
     for str_path in os.listdir(repo_dir):
-        if 'cookiecutter' in str_path and '{{' in str_path and '}}' in str_path:
+        if (
+            'cookiecutter' in str_path
+            and environment.variable_start_string in str_path
+            and environment.variable_end_string in str_path
```
Comment (kurtmckee) [inline cookiecutter/find.py:26]: This, and the similar change in `generate.py`, are the crucial elements of this bugfix. Nice!

**Review by kurtmckee — APPROVED** [review]
@mariovagomarzal Thanks for re-triggering CI after the linting PR was merged, and for adding a test to demonstrate this is fixed!

@audreyfeldroy @jensens @tmeckel Please merge this PR to close #1736. Thanks!

**Comment by jensens** [comment]
This changes the signature of the functions -> breaking change.

**Comment by ericof** [comment]
As this is a breaking change, we will add it to the 3.0.0 milestone

**Comment by kurtmckee** [comment]
Would it be sufficient to give the `environment` parameter a default of `None` and then conditionally use its settings? For example:

```python
def find(x, environment = None):
    left_delimiter = "{{"
    right_delimiter = "}}"
    if environment is not None:
        left_delimiter = environment.x
        right_delimiter = environment.y
    ...
```

Would that allow this to get merged immediately as a bugfix instead of a breaking change?

**Comment by jensens** [comment]
Was this addressed with merge of #1997 ?

**Comment by mariovagomarzal** [comment]
Yes, I think we can close this pull request.

Sorry for the late response.

**Pull request closed without merging**